index.html

```diff
@@ -148,6 +148,35 @@ <h2>Skills</h2>
         </ul>
     </li>
   </ul>
+  <footer>
+     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+     <form action="https://formspree.io/f/mayvbgnj" method="POST" class="contact-form">
+        <input type="text" placeholder="Full Name" class="text" maxlength="30" required>
+        <input type="email" class="text" placeholder="Email address" pattern="/^[a-zA-Z0-9.!#$%&amp;'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/" required>
```

Comment:
- [ ] Kindly note that your form is not submitted because the pattern you used here is not matching the correct email. Kindly fix this. here is the error I am getting :point_down: 

![image](https://user-images.githubusercontent.com/65068771/159675142-a76c736e-f933-441c-9536-ed7528f02c1e.png)